<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbName">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>42</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPersonType">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>82</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTelNo">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>125</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tel No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTelNo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInputBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSex">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>45</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sex</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMemoStr">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>165</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Memo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMemo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClearBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbM">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbW">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPersonType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTelNo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbAdding">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Adding Contacts</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFinding">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Finding Contacts</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblPhoneList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>34</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDelBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
